.travis.yml

```diff
@@ -16,24 +25,19 @@ install:
   - 'echo "Apache Ant ARCHIVE: $APACHE_ANT_BASE"'
   - '[ "${TRAVIS_OS_NAME}" = "linux" ] && wget http://apache.mirror.iphh.net/ant/binaries/$APACHE_ANT_BASE-bin.tar.gz && tar xzf $APACHE_ANT_BASE-bin.tar.gz && sudo mv $APACHE_ANT_BASE /usr/local/$APACHE_ANT_BASE && sudo rm -f /usr/local/ant && sudo ln -s /usr/local/$APACHE_ANT_BASE /usr/local/ant && sudo ln -s /usr/local/$APACHE_ANT_BASE/bin/ant /usr/local/bin/ant || true'
   - '[ "${TRAVIS_OS_NAME}" = "linux" ] && sudo apt-get -y install texinfo || true'
-  - '[ "${TRAVIS_OS_NAME}" = "osx" ] && brew update || true'
-  - '[ "${TRAVIS_OS_NAME}" = "osx" ] && brew uninstall libtool && brew install libtool || true'
-  - '[ "${TRAVIS_OS_NAME}" = "osx" ] && brew install ant || true'
+  # - '[ "${TRAVIS_OS_NAME}" = "osx" ] && brew update || true'

```

Comment:
Not sure we need to keep the old commented config lines. Totally your call.

Reply:
I initially thought that; also, if we only have the Linux ARM test we ould actually remove the `[ "${TRAVIS_OS_NAME}" = "linux" ]` conditional as well.  My thought was to leave them here in case Travis offers something (e.g., M1 hardware) that's not available on Github Actions, which would make adding the `osx` option back less error-prone.  So my preference is to comment out for now, and see which CI system offers different hardware options first.